recipes/blake3/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "blake3" %}
+{% set version = "0.1.7" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 99d688bd1b673de332a2927665e0e82eefae79b18b7c03564c5907d716e473b8
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
```

Nowadays there is a compiler macro for rust.
```suggestion
  build:
   - {{ compiler('rust') }}
```